digital_comms/ccam.py

```diff
@@ -223,9 +348,10 @@ def threshold_demand(self, SERVICE_OBLIGATION_CAPACITY):
 
     @property
     def demand(self):
-        """Estimate total demand based on population and penetration
+        """obj: The demand in capacity per km^2
```

Comment:
My concern is that if this is km^2, why do we then divide by area again in the above object? Need to double check this.

Reply:
Ok I've added a Todo